VungleAdapter/src/main/java/com/chartboost/mediation/vungleadapter/VungleAdapter.kt

```diff
@@ -664,153 +527,227 @@ class VungleAdapter : PartnerAdapter {
                 }
             }
 
-            Vungle.loadAd(
-                request.partnerPlacement,
-                adm,
-                adConfig,
-                object : LoadAdCallback {
-                    override fun onAdLoad(id: String) {
-                        PartnerLogController.log(LOAD_SUCCEEDED)
-                        resumeOnce(
-                            Result.success(createPartnerAd(null, request))
-                        )
-                    }
+            fun loadVungleFullScreenAd(fullscreenAd: () -> BaseFullscreenAd) {
```

Comment:
I guess I also question why this is a function. also why this is a lambda and not just a reference.

Reply:
Preference at this point, we can opt with just a reference.